docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform-2022.md

```diff
@@ -0,0 +1,225 @@
+  connection: {
+    client: 'postgres',
+    connection: {
+      host: env('DATABASE_HOST'), 
+      port: env.int('DATABASE_PORT'), 
+      database: env('DATABASE_NAME'), 
+      user: env('DATABASE_USERNAME'), 
+      password: env('DATABASE_PASSWORD'),
+      ssl: {
+        rejectUnauthorized:env.bool('DATABASE_SSL_SELF', false),
+      },
+    },
+    debug: false,
+  },
+});
+
+```
+
+</code-block>
+
+<code-block title='TYPESCRIPT'>
+
+```jsx
+//path:config/env/production/database.ts
+
+export default ({ env }) => ({
+  connection: {
+    client: 'postgres',
+    connection: {
+      host: env('DATABASE_HOST'), 
+      port: env.int('DATABASE_PORT'), 
+      database: env('DATABASE_NAME'), 
+      user: env('DATABASE_USERNAME'), 
+      password: env('DATABASE_PASSWORD'),
+      ssl: {
+        rejectUnauthorized:env.bool('DATABASE_SSL_SELF', false),
+      },
+    },
+    debug: false,
+  },
+});
+
+```
+
+</code-block>
+</code-group>
+
+### Configuring server
+
+<code-group>
+
+<code-block title='JAVASCRIPT'>
+
+```jsx
+//path:config/env/production/server.js
+
+module.exports = ({ env }) => ({
+    proxy: true,
+    url: env('APP_URL'),
+    app: { 
+      keys: env.array('APP_KEYS')
+    },
+  })
+
+```
+
+</code-block>
+
+<code-block title='TYPESCRIPT'>
+
+```jsx
+//path:config/env/production/server.ts
+
+export default ({ env }) => ({
+    proxy: true,
+    url: env('APP_URL'),
+    app: { 
+      keys: env.array('APP_KEYS')
+    },
+  })
+
+```
+
+</code-block>
+</code-group>
+
+### Adding postgres dependencies
+
+Connecting a postgres database to Strapi requires a set of Node modules contained in the `pg` package. Use the same package manager used to create the Strapi application to install `pg`. Run the following command in a terminal:
+
+<code-group>
+
+<code-block title="NPM">
+```bash
+npm install pg
+```
+</code-block>
+
+<code-block title="YARN">
+```bash
+yarn add pg
+```
+</code-block>
+
+</code-group>
+
+### Committing the project to a remote repository
+
+Save the Strapi application locally then, in a terminal, run :
+
+1. `git add .`
+2. `git commit -m "commit message"`
+3. `git push`
+
+## Creating a DigitalOcean App
+
+On the DigitalOcean website click on the **Create** button and select *Apps*.  Next select GitHub and authorize access to the correct repository. Next:
+
+1. select the branch,
+2. select the source directory (optional),
+3. chose whether or not to "Autodeploy" when an update is pushed to the GitHub repository,
+4. click the **Next** button.
+
+### Connecting a web app to a database
+
+On the next screen:
+
+1. click **add resource**,
+2. select database, and click **Add**,
+3. On the next screen select *dev database* and name the database. The default database name is "db" and that will be used in the subsequent sections.
+4. Click the **Create and attach** button.
+5. On the next screen click the **Next** button to set the environment variables.
+
+### Environmental variables
```

Comment:
This title should start with an action verb so that the user knows what they should do here + to make that title fit with the other titles of the documentation.

Reply:
Also, this section needs restructuring I think. We have some steps here and there which are not in proper procedures but lot in big paragraphs ; and it's difficult to find what is the action/goal of that section.

Reply:
- Changed the title to **Add environment variables**
- edited the procedure into a numeric step form
- clarified some of the details
- added a callout about secret generation